doc/ref_model/chapters/chapter06.md

```diff
@@ -98,69 +98,63 @@ ETSI ETSI (Ref: NFV IFA 019 v03101p) has defined a set of technology independent
-| |||Input|accChannel|Channel where event is requested for |
-| AccDataAvailableEventNotificationRequest|AccDataAvailableEventNotificationResponse|NFVI → VNF|Input|regHandlerId|Reference used by VNF when registering for the event |
-| DeRegisterForAccDataAvailableEventRequest|DeRegisterForAccDataAvailableEventResponse|VNF → NFVI|Input|accChannel|Channel related to the event  |
-| AllocateAccResourceRequest|AllocateAccResourceResponse|VIM → NFVI|Input|attachTargetInfo|the resource the |
-| accelerator is to be attached to (e.g., VM) |
-| |||Input|accResourceInfo|Accelerator Information |
-| |||Output|accResourceId|Id if successful |
-| ReleaseAccResourceRequest|ReleaseAccResourceResponse|VIM → NFVI|Input|accResourceId|Id of resource to be released |
-| QueryAccResourceRequest|QueryAccResourceResponse|VIM → NFVI|Input|hostId|Id of specified host |
-| |||Input|Filter|Specifies the accelerators for which query applies |
-| |||Output|accQueryResult|Details of the accelerators matching the input filter located in the selected host. |
-| GetAccStatisticsRequest|GetAccStatisticsResponse|VIM → NFVI|Input|accFilter|Accelerator subsystems from which data is requested |
-| |||Input|accStatSelector|attributes of AccStatistics whose data will be returned |
-| |||Output|accStatistics|Statistics data of the accelerators |
-| matching the input filter located in the |
-| selected host. |
-| ResetAccStatisticsRequest|ResetAccStatisticsResponse|VIM → NFVI|Input|accFilter|Accelerator subsystems for which data is to be reset |
-| |||Input|accStatSelector|attributes of AccStatistics whose data will be reset |
-| SubscribeRequest|SubscribeResponse|VIM → NFVI|Input|hostId|Id of specified host |
-| |||Input|Filter|Specifies the accelerators and related alarmsThe filter could include accelerator information, severity of the alarm, etc. |
-| |||Output|SubscriptionId|Identifier of the successfully created |
-| subscription. |
-| UnsubscribeRequest|UnsubscribeResponse|VIM → NFVI|Input|hostId|Id of specified host |
-| |||Input|SubscriptionId|Identifier of the subscription to be |
-| unsubscribed. |
-| Notify||NFVI → VIM|||NFVI notifies an alarm to VIM |
-| GetAlarmInfoRequest|GetAlarmInfoResponse|VIM → NFVI|Input|hostId|Id of specified host |
-| |||Input|Filter|Specifies the accelerators and related alarmsThe filter could include accelerator information, severity of the alarm, etc. |
-| |||Output|Alarm|Information about the alarms if filter matches an alarm. |
-|  |
-| AccResourcesDiscoveryRequest|AccResourcesDiscoveryResponse|VIM → NFVI|Input|hostId|Id of specified host |
-| |||Output|discoveredAccResourceInfo|Information on the acceleration resources discovered within the NFVI. |
-| OnloadAccImageRequest|OnloadAccImageResponse|VIM → NFVI|Input|accResourceId|Identifier of the chosen accelerator in the NFVI. |
-| |||Input|accImageInfo|Information about the acceleration image. |
-| |||Input|accImage|The binary file of acceleration image. |
+<table>
```

Comment:
Any particular reason for shifting to html table from markdown?

Reply:
@deebhatia HTML looks better in general and allows you to have merged cells. for complex tables with many entires, I think use plain HTML (no fancy CSS etc) should be ok. do you agree?

Reply:
Okay.